etna/analysis/plotters.py

```diff
@@ -1121,6 +1146,8 @@ def plot_feature_relevance(
         TSDataset with timeseries data
     relevance_table:
         method to evaluate the feature relevance
+        RelevanceTable uses relevance table with feature importance from model
```

Comment:
Here probably `ModelRelevanceTable` is meant.

According to the [rendered documentation](https://deploy-preview-795--etna-docs.netlify.app/api/etna.analysis.plotters.plot_feature_relevance.html?highlight=plot_feature_relevance#etna.analysis.plotters.plot_feature_relevance) (you can find the link in conversation of this PR), this part is rendered in a single line, it is not nice.

Read in our contribution guide about formatting documentation according to ReST markup language.

I would suggest to write it like this:
```
relevance_table:
        method to evaluate the feature relevance;

        * if :py:class:`~etna.analysis.feature_relevance.relevance.StatisticsRelevanceTable` table is used then relevances are normalized p-values

        * if :py:class:`~etna.analysis.feature_relevance.relevance.ModelRelevanceTable` table is used then relevances are importances from some model

```